pages/actions/pause-audio.md

```diff
@@ -0,0 +1,78 @@
+# pauseAudio
+
+pauseAudio action allows users to pause a already playing audio file
```

Comment:
@hemish11 let's make sure the intros adhere to proper English grammar and clarity, like:

> The pauseAudio action allows users to pause an already-playing audio file.